samples/mac/weather/MyManagedStuff/MyManagedStuff/WeatherFetcher.cs

```diff
@@ -0,0 +1,50 @@
+﻿﻿using System;
+using System.Json;
+using System.Net;
+
+namespace MyManagedStuff {
```

Comment:
Since you control the _contract_ you can remove the namespace `MyManagedStuff` and replace it with a 3 letter prefix to the type (e.f. `XAM` ?). That will make the sample nicer looking for ObjC developers.